tensorboard/plugins/debugger_v2/tf_debugger_v2_plugin/store/debugger_types.ts

```diff
@@ -487,6 +487,19 @@ export interface DebuggerState {
   runs: DebuggerRunListing;
   runsLoaded: LoadState;
 
+  // Timestamp for the last data polling (including the
+  // initial data-loading event triggered by plugin loading,
+  // and those triggered by TensorBoard's core auto and manual
+  // reloading).
+  lastDataPollTime: number;
+  // Timestamp for the last non-empty data polling result.
+  // This is updated when any of the following is refreshed:
+  // - activeRunId
+  // - executions.executionDigestsLoaded.numExecutions
+  // - graphExecutions.executionDigestsLoaded.numExecutions
+  // - alerts.alertsLoaded.numAlerts
+  lastNewPollDataTime: number;
```

Comment:
Upon more careful inspection of the reducer code, this literally means `lastNonEmptyDataLoadedTimeInMs`. Can we just say that?

Reply:
This is now called `lastNonEmptyPollDataTimeMs`, to be slightly less verbose.